### ISSUE TITLE
Add support for token_affiliation

### ISSUE THREAD

**Issue by DanielMalmgren** [MarcelCoding/jitsi-openid#114]
I will be needing support for [token affiliation](https://github.com/jitsi-contrib/prosody-plugins/tree/main/token_affiliation) in my jwt's. It would mean that if the incoming openid token contains an attribute named "affiliation" I would simply need jitsi-openid to put it in the jwt sent to Jitsi. That would be an easy fix, right?

**Comment by DanielMalmgren** [comment]
I created a pr for this: #115

**Comment by DanielMalmgren** [comment]
No, I'm sorry, but I think we broke it completely now... I get the following error whatever I try to do:

> Missing request extension: Extension of type `openidconnect::Client<openidconnect::claims::EmptyAdditionalClaims, openidconnect::core::CoreAuthDisplay, openidconnect::core::CoreGenderClaim, openidconnect::core::CoreJweContentEncryptionAlgorithm, openidconnect::core::CoreJwsSigningAlgorithm, openidconnect::core::jwk::CoreJsonWebKeyType, openidconnect::core::jwk::CoreJsonWebKeyUse, openidconnect::core::jwk::CoreJsonWebKey, openidconnect::core::CoreAuthPrompt, oauth2::StandardErrorResponse<oauth2::basic::BasicErrorResponseType>, oauth2::StandardTokenResponse<openidconnect::id_token::IdTokenFields<openidconnect::claims::EmptyAdditionalClaims, oauth2::EmptyExtraTokenFields, openidconnect::core::CoreGenderClaim, openidconnect::core::CoreJweContentEncryptionAlgorithm, openidconnect::core::CoreJwsSigningAlgorithm, openidconnect::core::jwk::CoreJsonWebKeyType>, oauth2::basic::BasicTokenType>, oauth2::basic::BasicTokenType, oauth2::StandardTokenIntrospectionResponse<oauth2::EmptyExtraTokenFields, oauth2::basic::BasicTokenType>, oauth2::revocation::StandardRevocableToken, oauth2::StandardErrorResponse<oauth2::revocation::RevocationErrorResponseType>>` was not found. Perhaps you forgot to add it? See `axum::Extension`.

**Comment by MarcelCoding** [comment]
Sorry my bad, fixed in https://github.com/MarcelCoding/jitsi-openid/commit/29747a75ef0bd93b5ff8db4dc77fcee05981e954.

**Comment by DanielMalmgren** [comment]
> fixed in [29747a7](https://github.com/MarcelCoding/jitsi-openid/commit/29747a75ef0bd93b5ff8db4dc77fcee05981e954)

Works like a charm now! Thanks a lot!
